README.md

```diff
@@ -137,11 +136,11 @@ disable=raw-checker-failed,
 Re-run the `make all` command. Now all errors are fixed or suppressed and the build is successful.
```

Comment:
in line 121, editing options for .pylintrc file. The section is called `MESSAGES CONTROL`. There is an `S` missing. which makes a bit harder to find it when doing `ctrl + f` to edit the section.